spec/enumerable_spec.rb

```diff
@@ -0,0 +1,183 @@
+# spec/enumerable_spec.rb
+
+# rubocop : disable Layout/LineLength
+
+require_relative '../enumerable.rb'
+
+describe Enumerable do
+  describe '#my_each' do
+    let(:new_arr) { [2, 6, 8] }
+
+    it 'Iterate over the array is called and do the instructions in the block predicate over each element' do
+      expect do
+        new_arr.my_each { |el| puts el + 2 }
+      end.to output("4\n8\n10\n").to_stdout
+    end
```

Comment:
lets compare the values with the main enumerables ,thats a proper way of testing .for example
`expect( new_arr.my_each { |el| puts el + 2 }).to eql( new_arr.each { |el| puts el + 2 }) `

#### Kindly  make changes to all codes like this